common/static/common/js/spec/components/feedback_spec.js

```diff
@@ -1,387 +1,383 @@
-// Generated by CoffeeScript 1.6.1
```

Comment:
Could someone help me understand the diff here? I assume it has do do with moving from coffeescript to javascript.

Was there a change other than re-indenting with 4 spaces? I'd prefer to see 2 spaces preserved, as it better reflects the history of this file when blaming.